src/routes/view-space.js

```diff
@@ -82,6 +82,42 @@ const SurveyJumbo = (props) => (
     </Jumbotron>
 );
 
+/**
+ * Improvements:
+ *  - animated heart icon for the favorites button (see: https://codepen.io/mapk/pen/ZOQqaQ)
+ *  - better background color
+ *  - include a toast (maybe "Added to Favorites") once clicked
+ *  - disable button as soon as it's clicked instead of having to refresh the page
+ *  - favorites button on the main nav bar should automatically requery for the
+ *    local storage data instead of having to do a manual refresh
+ */
```

Comment:
another improvement -> user can unfavorite immediately after they favorite. or like, any unfavoriting capability on this page